### PR TITLE
Improve fix-cs command in Makefile

### LINKED ISSUE #1: tests folder doesn't exist for cs fixer
Hello, 

I'm using this template and my symfony configuration is as basic as you configured in `composer.json`, so `/tests` folder is not included but your cs fixer in `Makefile` is extended to `/tests` folder also. 

The error message is ` The path "tests/" is not readable.`

Is there any way to not include if sf5 config doesn't include `/tests` or to check folders to fix dynamically ? 

Thanks :)

### PR DESCRIPTION
**Pull request by hwaseonchoi**
Fix https://github.com/adaniloff/symfony-bootstrap/issues/1

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -73,5 +73,8 @@ help: ## Outputs this help screen
 	fi
 
 fix-cs: .php_cs ## Run cs-fixer
-	$(CMD) $(CS_FIXER) fix src/ tests/ --config=.php_cs
-
+ifneq "$(COMMAND_ARGS)" ""
+	$(CMD) $(CS_FIXER) fix $(COMMAND_ARGS)/ --config=.php_cs
```
Comment (adaniloff) [inline Makefile]:
```suggestion
	$(CMD) $(CS_FIXER) fix $(COMMAND_ARGS) --config=.php_cs
```

File: Makefile [hunk 2]
```diff
@@ -73,5 +73,8 @@ help: ## Outputs this help screen
...
+else
+	@echo 'please specify directory you want to fix as follow: make fix-cs src'
```
Comment (adaniloff) [inline Makefile]:
```suggestion
	@echo 'Please specify the directories in which you want to run the fixer. For example: "make fix-cs src/ tests/".'
```

**Review by adaniloff — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (hwaseonchoi) [reply Makefile]: Done

File: Makefile (hunk 2 above)
Comment (hwaseonchoi) [reply Makefile]: Done, thanks !

**Review by adaniloff — APPROVED** [review]

**Pull request merged**